### PR TITLE
Update spacing and alignment on VotingWidget

### PR DESCRIPTION
**Pull request by ArmandoGraterol**
* Increased space between VotingWidget title and voting results
* Added `drop-shadow` to voter avatars
* Updated border of voter avatars

FireShot Capture 352 - Colony - localhost

Resolves DEV-383

### CODE REVIEW

**Review by alicjakujawa — APPROVED** [review]
Screenshot 2021-06-08 at 10 13 49
One more thing, space between progress bar and avatars is a bit too big (in compare do designs) after you update it, its good to merge

**Review by chinins — APPROVED** [review]
Good to go without changing the distance to the avatar - in my opinion (see me separate comment).

**Comment by chinins** [comment]
> Screenshot 2021-06-08 at 10 13 49
> 
> One more thing, space between progress bar and avatars is a bit too big (in compare do designs) after you update it, its good to merge

@alicjakujawa, I think that in figma the space is smaller because there are several avatars and it will be the case in this instance. When there is only one, it's normal/ok to have it larger - so that the bar doesn't change length depending on how many avatars we have... 

Let me know what you both think.

**Comment by ArmandoGraterol** [comment]
> > Screenshot 2021-06-08 at 10 13 49
> > One more thing, space between progress bar and avatars is a bit too big (in compare do designs) after you update it, its good to merge
> 
> @alicjakujawa, I think that in figma the space is smaller because there are several avatars and it will be the case in this instance. When there is only one, it's normal/ok to have it larger - so that the bar doesn't change length depending on how many avatars we have...
> 
> Let me know what you both think.

@chinins You are correct! I just tested it with 3 avatars, and the space between the avatars and the progress bar is small as in the design. So I think it's good as it is. 

I will not merge this tho as I think there's still discussion on whether the avatars should be aligned to the progress bar instead of the icon, so I'll wait.

**Pull request merged**
